content/en/docs/setup/platform-setup/kops/index.md

```diff
@@ -16,27 +16,24 @@ If you wish to run Istio [Secret Discovery Service](https://www.envoyproxy.io/do
     $ kops edit cluster $YOURCLUSTER
     {{< /text >}}
 
-1. Add the following in the configuration file:
+2. Add the following in the configuration file:
```

Comment:
Leaving these items as 1. lets them be renumbered automatically as other items are added in the middle.

Reply:
thanx for catching this ^^ pushed changes